pygradle-plugin/src/main/groovy/com/linkedin/gradle/python/extension/PythonVersion.java

```diff
@@ -21,6 +21,10 @@
 
     private final String version;
 
+    public static String[] whitelistedPythonVersions = { "2.6", "2.7", "3.4", "3.5", "3.6" };
```

Comment:
You probably want to make this a `Collection<String>` (probably actually a Set internally), so that you can add and remove from it without having to deal with arrays.